### DISCUSSION TITLE
How to invoke magic commands in Thonny on Windows

### DISCUSSION THREAD

**Discussion by Sonoton** [thonny/thonny#3268] [General]
Are magic commands (%dir, %cd , etc) available on thonny (Windows installed).
Is it possible to create an env environment from File-> New -> Create Env somehow in thonny (Windows installed).

**Comment by Sonoton** [comment]
I have actually only been able to use magic commands by launching !juypter lab from the thonny console, then using magic commands (full list) in jupyter.  I think the help file in thonny needs to be corrected to reflect that otherwise, magic commands are not usuable in thonny, (other than as shortcuts of the menu items, as stated in the thonny help section).

**Comment by aivarannamaa** [comment]
Thonny has some magic commands, which are inspired by IPython, but it is not technically related to IPython.
